app/assets/javascripts/discourse/models/rest.js.es6

```diff
@@ -30,6 +30,7 @@ const RestModel = Ember.Object.extend({
 
         self.setProperties(payload);
         self.afterUpdate(res);
+        res.target = self;
```

Comment:
This seems reasonable to me, but we need to add a test for this functionality.

Reply:
Adding tests for that right now. Currently, they look like this:

```js
promise.then(function(result) {
  assert.ok(result.target, "it has a reference to the record");
  // ...
});
```

While I’m at it, is there something more I should test for here?